phpunit/functional/DBTest.php

```diff
@@ -377,7 +377,7 @@ public function testTablesHasItemtype()
         foreach ($list as $line) {
             $this->assertCount(1, $line);
             $table = $line['TABLE_NAME'];
-            if (in_array($table, $excluded_tables, true)) {
+            if (in_array($table, $excluded_tables, true) || strpos($table, 'glpi_plugin_formcreator_') === 0) {
```

Comment:
Do you still need this ?
No idea just wondering because I think the formcreator SQL dump is no longer included by default (or at least I heard some discussions about that) so maybe this check is no longer relevant ?